tests/storage/business/test_xpansion_manager.py

```diff
@@ -69,129 +52,98 @@ def make_link(link_manager: LinkManager, study: StudyInterface) -> None:
 
 
-@pytest.fixture
-def empty_study_810(tmp_path: Path) -> StudyInterface:
-    return make_empty_study(tmp_path, 810)
-
-
-@pytest.fixture
-def empty_study_880(tmp_path: Path) -> StudyInterface:
-    return make_empty_study(tmp_path, 880)
-
-
 @pytest.mark.unit_test
-@pytest.mark.parametrize(
-    "version, expected_output",
-    [
-        (
-            810,
-            {
-                "candidates": {},
-                "capa": {},
-                "constraints": {},
-                "sensitivity": {"sensitivity_in": {}},
-                "settings": {
-                    "master": "integer",
-                    "uc_type": "expansion_fast",
-                    "optimality_gap": 1,
-                    "relative_gap": 1e-06,
-                    "relaxed_optimality_gap": 1e-05,
-                    "max_iteration": 1000,
-                    "solver": "Xpress",
-                    "log_level": 0,
-                    "separation_parameter": 0.5,
-                    "batch_size": 96,
-                    "timelimit": int(1e12),
-                },
-                "weights": {},
-            },
-        ),
-    ],
-)
 def test_create_configuration(
-    xpansion_manager: XpansionManager, tmp_path: Path, version: int, expected_output: JSON
+    xpansion_manager: XpansionManager,
+    tmp_path: Path,
+    empty_study_810: FileStudy,
 ) -> None:
     """
     Test the creation of a configuration.
     """
-    empty_study = make_empty_study(tmp_path, version)
-
+    empty_study = FileStudyInterface(empty_study_810)
     with pytest.raises(ChildNotFoundError):
         empty_study.get_files().tree.get(["user", "expansion"], expanded=True, depth=9)
 
     xpansion_manager.create_xpansion_configuration(empty_study)
 
     actual = empty_study.get_files().tree.get(["user", "expansion"], expanded=True, depth=9)
-    assert actual == expected_output
+    assert actual == {
+        "candidates": {},
+        "capa": {},
+        "constraints": {},
+        "sensitivity": {"sensitivity_in": {}},
+        "settings": {
+            "master": "integer",
+            "uc_type": "expansion_fast",
+            "optimality_gap": 1,
+            "relative_gap": 1e-06,
+            "relaxed_optimality_gap": 1e-05,
+            "max_iteration": 1000,
+            "solver": "Xpress",
+            "log_level": 0,
+            "separation_parameter": 0.5,
+            "batch_size": 96,
+            "timelimit": int(1e12),
+        },
+        "weights": {},
+    }
 
 
 @pytest.mark.unit_test
-def test_delete_xpansion_configuration(xpansion_manager: XpansionManager, tmp_path: Path) -> None:
+def test_delete_xpansion_configuration(
+    xpansion_manager: XpansionManager, tmp_path: Path, empty_study_810: FileStudy
+) -> None:
     """
     Test the deletion of a configuration.
     """
-    study = make_empty_study(tmp_path, 810)
-
+    empty_study = FileStudyInterface(empty_study_810)
```

Comment:
actually I think just `study` was a better name: the study does not remain empty for long (same in existing tests though, tha could be part of another PR)